src/com/ableton/Pipenv.groovy

```diff
@@ -0,0 +1,114 @@
+package com.ableton
+
+
+/**
+ * Provides an easy way to run a command with Pipenv using multiple Python versions.
+ */
+class Pipenv implements Serializable {
+  /**
+   * Script context.
+   * <strong>Required value, may not be null!</strong>
+   */
+  @SuppressWarnings('FieldTypeRequired')
+  def script
+
+  /**
+   * Run a command with Pipenv using multiple versions of Python. Because the virtualenv
+   * created by Pipenv must be wiped out between runs, this function cannot be
+   * parallelized and therefore the commands are run serially for each Python version.
+   *
+   * @param args Map of arguments. Valid arguments include:
+   *             <ul>
+   *               <li>
+   *                 {@code command}: Command to run.
+   *                 <strong>Required value, may not be null!</strong>
+   *               </li>
+   *               <li>
+   *                 {@code pythonVersions}: List of Python versions to run the command
+   *                 with. This argument is passed to Pipenv via {@code pipenv --python}.
+   *                 See {@code pipenv --help} for supported syntax.
+   *                 <strong>Required value, may not be null!</strong>
+   *               </li>
+   *               <li>
+   *                 {@code returnStatus}: Return the status code of the script.
+   *                 <strong>Note:</strong> this option and the {@code returnStdout}
+   *                 option are mutually exclusive.
+   *               </li>
+   *               <li>
+   *                 {@code returnStatus}: Return standard output from the script.
+   *                 <strong>Note:</strong> this option and the {@code returnStatus}
+   *                 option are mutually exclusive.
+   *               </li>
+   *             </ul>
+   * @return Map of return values. The keys in the map correspond to the Python versions
+   *         given in {@code args.pythonVersions}, and the values are either integers,
+   *         strings, or null, depending on whether {@code returnStatus},
+   *         {@code returnStdout}, or neither option was specified (respectively).
+   */
+  Map runCommand(Map args) {
```

Comment:
Why do we need both `runCommand` and `runWith`? `runCommand` seems to be a subset of `runWith`.

Reply:
It's a convenience, but I'm open to removing `runCommand` and only having `runWith`.